completions/ccache

```diff
@@ -26,8 +29,11 @@ _ccache()
 
     $split && return
 
-    COMPREPLY=( $( compgen -W '$( _parse_help "$1" )' -- "$cur" ) )
-    [[ $COMPREPLY == *= ]] && compopt -o nospace
+    if [[ "$cur" == -* ]]; then
+        COMPREPLY=( $( compgen -W '$( _parse_help "$1" )' -- "$cur" ) )
+        [[ $COMPREPLY == *= ]] && compopt -o nospace
+        return
+    fi
```

Comment:
Like the previous hunk, this seems unrelated to the actual fix, they look mostly like cosmetic changes to me. Or am I missing something? If not, please leave these out of the patch.